src/component/mxgraph/GraphCellUpdater.ts

```diff
@@ -66,33 +66,33 @@ export default class GraphCellUpdater {
   }
 
   addOverlays(bpmnElementId: string, overlays: Overlay | Overlay[]): void {
-    const mxCell = this.graph.getModel().getCell(bpmnElementId);
-    if (!mxCell) {
+    const cell = this.graph.getModel().getCell(bpmnElementId);
+    if (!cell) {
```

Comment:
**thought:** I intially planned to do this change in a dedicated PR, but as it is very limited, I finally decided to do it here (with all other related mxCell renames)